pkg/component/kubernetes/controllermanager/controllermanager.go

```diff
@@ -868,10 +774,6 @@ func (k *kubeControllerManager) computeResourceRequirements(ctx context.Context)
 		},
 	}
 
-	if k.values.HVPAConfig == nil || !k.values.HVPAConfig.Enabled {
```

Comment:
Do we need to keep the method `computeResourceRequirements` at all? This was here, because HVPA changed the Deployment's resources directly, but with VPA this doesn't really matter.
I think we can just inline the resources into the Deployment and remove this method?

Reply:
The reason to keep it now, is a bit different from the one for its original existence. I kept it because I'm concerned about about the following chain of events on a large shoot:
- Shoot reconciliation resets Deployment requests from HVPA-adjusted level to original level.
- Deployment change causes pods to roll.
- Pod creation catches new VPA instance while is snapshot is still empty. VPA sets low requests.
- OOM

On a related note, if we keep this temporarily, I wonder which release we should target for the removal of the code.  Skipping just one release seems ok (I can't imagine how a disruption-sensitive cluster would miss reconciliation for the duration of an entire release cycle), but just in case, maybe skip 2?

Reply:
Per offline discussion, will use the same transition period, as for the rest of the temp HVPA code (drop in 1.97).

Reply:
As discussed offline: if there are good reasons to keep this, then please add a TODO to remove this with v1.97 as above